src/scripts/menus/help.js

```diff
@@ -1,21 +1,30 @@
 import { remote } from 'electron';
+import servers from '../servers';
 const APP_NAME = remote.app.getName();
 
 const helpTemplate = [
     {
         label: APP_NAME + ' Help',
-        click: function () {
-            remote.shell.openExternal('https://rocket.chat/docs');
-        }
+        click: () => remote.shell.openExternal('https://rocket.chat/docs')
+    },
+    {
+        type: 'separator'
+    },
+    {
+        label: 'Report Issue',
```

Comment:
I set the link to the main repository, as most issues people report are things with the web app, but I was thinking we could potentially have a landing page on rocket.chat which will ask the user if the issue is related to. We could even send all the app details in the url (e.g. desktop version, rc version) and send them to github with a pre populated issue (https://github.com/RocketChat/Rocket.Chat.Electron/issues/new?title=foo&body=bar)